src/main/java/com/ullink/IKVMDocLet.java

```diff
@@ -1,903 +1,903 @@
-     * 
-     * @param args the program arguments
-     */
-    public static void main(String[] args) {
-        Main.execute("ikvmdoc", IKVMDocLet.class.getName(), args);
-    }
-
-    /**
-     * Check for ikvmdoc specific options. Returns the number of arguments that must be specified on the command
-     * line for the given option. For example, "-assembly IKVM.OpenJDK.ClassLibrary.dll" would return 2.
-     * 
-     * @param option the option to evaluate and return the number of arguments for 
-     * @return number of arguments on the command line for an option including the option name itself.
-     *   Zero return means option not known. Negative value means error occurred.
-     */
-    public static int optionLength(String option) {
-        if (ASSEMBLY_PARAMETER.equals(option)) {
-            return 2;
-        } else if (STRICT_FINAL_FIELD_SEMANTICS_PARAMETER.equals(option)) {
-            return 1;
-        } else if (HTML_PARAMETER.equals(option)) {
-            return 1;
-        } else if (AUTHOR_PARAMETER.equals(option)) {
-            return 1;
-        } else if (DEPRECATED_PARAMETER.equals(option)) {
-            return 1;
-        } else if (SINCE_PARAMETER.equals(option)) {
-            return 1;
-        } else if (VERSION_PARAMETER.equals(option)) {
-            return 1;
-        }
-        // allow standard options, although they're ignored (for now) 
-        return ConfigurationImpl.getInstance().optionLength(option);
-    }
-    
-    /**
-     * Check that ikvmdoc options have the correct arguments.
-     * 
-     * @param options the options to check
-     * @param reporter the error reported used to report any failures to
-     * @return <code>true</code> if the options are valid;
-     *   <code>false</code> if the options are invalid
-     */
-    public static boolean validOptions(String[][] options, DocErrorReporter reporter) {
-        ERROR_REPORTER = reporter;
-        
-        for (String[] option : options) {
-            if (ASSEMBLY_PARAMETER.equals(option[0])) {
-                ASSEMBLY_FILE = new File(option[1]);
-                
-                if (!ASSEMBLY_FILE.isFile() || !ASSEMBLY_FILE.exists()) {
-                    reporter.printError("The assembly file specified '" + ASSEMBLY_FILE.getAbsolutePath() + "' is invalid.");
-                    return false;
-                }
-            } else if (HTML_PARAMETER.equals(option[0])) {
-                OUTPUT_HTML = false;
-            } else if (AUTHOR_PARAMETER.equals(option[0])) {
-                OUTPUT_AUTHOR = true;
-            } else if (DEPRECATED_PARAMETER.equals(option[0])) {
-                OUTPUT_DEPRECATED = false;
-            } else if (SINCE_PARAMETER.equals(option[0])) {
-                OUTPUT_SINCE = false;
-            } else if (VERSION_PARAMETER.equals(option[0])) {
-                OUTPUT_VERSION = true;
-            }
-        }
-        
-        return true;
-    }
+package com.ullink;
```

Comment:
I am not sure what it was exactly but every time I checked out git modified these files.  So I guess it has something to do with line endings or encoding.